frontends/bsd/src/screens/ballot_eject_screen.tsx

```diff
@@ -441,49 +441,26 @@ export function BallotEjectScreen({
                 : 'Unknown Reason'}
             </EjectReason>
             <p>
-              This last scanned sheet <strong>was not tabulated</strong>.
+              This ballot <strong>has not yet been tabulated</strong>. What
+              would you like to do?
             </p>
             {allowBallotDuplication ? (
               <React.Fragment>
-                <h4>Original Ballot Scan</h4>
                 <p>
-                  Remove ballot and create a duplicate ballot for the Resolution
-                  Board to review.
-                  <br />
                   <Button
                     primary={ballotState === 'removeBallot'}
                     onPress={() => setBallotState('removeBallot')}
                   >
-                    Original Ballot Removed
+                    Don&apos;t Count It
                   </Button>
                 </p>
-                <h4>Duplicate Ballot Scan</h4>
                 <p>
-                  {isOvervotedSheet ? (
-                    <React.Fragment>
-                      Confirm ballot sheet was reviewed by the Resolution Board
-                      and tabulate as ballot sheet with an{' '}
-                      <strong>overvote</strong>.
-                    </React.Fragment>
-                  ) : isBlankSheet ? (
-                    <React.Fragment>
-                      Confirm ballot sheet was reviewed by the Resolution Board
-                      and tabulate as a <strong>blank</strong> ballot sheet and
-                      has no votes.
-                    </React.Fragment>
-                  ) : (
-                    <React.Fragment>
-                      Confirm ballot sheet was reviewed by the Resolution Board
-                      and tabulate as ballot with issue which could not be
-                      determined.
-                    </React.Fragment>
-                  )}
                   <br />
```

Comment:
Sanity check: Do we still need this break here, now that the text above has been removed?

Reply:
probably not :) Layout looked ok to me. Will happily do another tweak of course.